templates/compare/index.html

```diff
@@ -0,0 +1,289 @@
+        <p>Robustness is a key primitive, particularly for production environments. Transactional over-the-air patching and security fixes, provided through the snap package as well as self-healing high availability of the Kubernetes control plane services, give MicroK8s the edge for mission-critical workload support.</p>
+        <p>With MicroK8s you get a complete, CNCF-compliant Kubernetes environment and all the services you need to run your OCI containers at scale, such as networking & load balancing, storage, service mesh, observability, GPU & FPGA acceleration, multi-cluster management and more - something unique among lightweight Kubernetes distributions.</p>
+      </div>
+      <div class="col-5 u-align--center u-vertically-center u-hide--small u-hide--medium">
+        <img src="https://assets.ubuntu.com/v1/990738e2-kubernetes-cloud.svg" alt="" width="263" height="150" loading="lazy" />
+      </div>
+    </div>
+  </section>
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Kubernetes resources</h2>
+  </div>
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <div class="p-heading-icon--small">
+        <div class="p-heading-icon__header">
+          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/6e184942-Webinar.svg" alt="">
+          <h3 class="p-heading--4 p-heading-icon__title">Webinars on demand</h3>
+        </div>
+        <ul class="p-list">
+          <li class="p-list__item">
+            <a href="https://www.brighttalk.com/webcast/6793/378029?utm_source=brighttalk-portal&utm_medium=web&utm_content=microk8s&utm_campaign=webcasts-search-results-feed">An intro to MicroK8s</a>
+          </li>
+          <li class="p-list__item">
+            <a href="https://www.brighttalk.com/webcast/6793/488796/kubernetes-at-the-edge-easy-as-pi">K8s at the edge: easy as “Pi”
+            </a>
+          </li>
+          <li class="p-list__item">
+            <a href="https://www.brighttalk.com/webcast/6793/522563">Embedded Kubernetes for secure IoT Edge</a>
+          </li>
+          <li class="p-list__item">
+            <a href="https://www.brighttalk.com/webcast/6793/519512?utm_source=brighttalk-portal&utm_medium=web&utm_content=canonical&utm_term=search-result-7&utm_campaign=webcasts-search-results-feed">MicroK8s on IBM Z — minimal footprint meets zero downtime</a>
+          </li>
+        </ul>
+     </div>
+    </div>
+    <div class="col-4 p-divider__block">
+      <div class="p-heading-icon--small">
+        <div class="p-heading-icon__header">
+          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/6e184942-Webinar.svg" alt="">
+          <h3 class="p-heading--4 p-heading-icon__title">Webinars on demand</h3>
```

Comment:
@alxchalkias it looks a little odd with two headers on this section being identical - what do you think of doing this instead?
![Screenshot from 2022-01-26 14-34-26](https://user-images.githubusercontent.com/2376968/151182910-0a405686-aee2-4a6c-a8e0-e042ccc3c4dd.png)

Reply:
+1, this is much better